### ISSUE TITLE
Accessibility: Side Bar becomes Non-functional at 320X256 - Violates 1.4.10 Reflow WCAG 2.1 Requirement (Follow-up Issue Referencing Previously Closed Bug)

### ISSUE THREAD

**Issue by anitakumari8** [microsoft/vscode#285871]
This issue was raised previously in [#249951](https://github.com/microsoft/vscode/issues/249951). According to the discussion, it was reported as working for [@joaomoreno](https://github.com/joaomoreno). However, I’ve tested it again and the issue still does not work on my end.
Since my previous GitHub account [anitakumari28](https://github.com/anitakumari28) was deactivated, I’m creating this new issue to follow up on the original issue.

https://github.com/user-attachments/assets/6194b86b-de8a-478c-9174-4c5c6ed6d3d4
